### ISSUE TITLE
Update flex version check in configure.ac

### ISSUE THREAD

**Issue by guitorri** [Qucs/qucs#334]
Reported by Walter woods:
```
Also, I had to modify qucs-core/configure line 18186 (checking flex version) from:
>     2.5.9 | [2-9].[5-9].[1-9][0-9])
to
>     2.5.9 | [2-9].[5-9].[1-9][0-9]*)
Since flex --version reports "flex 2.5.35 Apple(flex-31)" on Mac OS X.
```

**Comment by felix-salfelder** [comment]
> qucs-core/configure

shouldn't that go to configure.ac?

apart from that, the change looks safe to me... note, these are shell patterns, not regexps, this confused me at first. need more coffee `;D`.

**Comment by guitorri** [comment]
Yes it should be in the configure.ac...

**Comment by guitorri** [comment]
Fix merged with #344.
